components/icon-with-link.js

```diff
@@ -1,16 +1,15 @@
 export const IconWithLink = ({ icon, link, color = '#009fb7' }) => {
   const IconComponent = icon;
   return (
-    <a href={link} className="icon-wrapper">
-      <IconComponent />
+    <a href={link} target="_blank">
```

Comment:
if we use target _blank her we need to ad also `rel="noopener noreferrer"` because of security problems.

Reply:
here is the security problem about target _blank

https://owasp.org/www-community/attacks/Reverse_Tabnabbing